alerts/state.libsonnet

```diff
@@ -27,7 +29,9 @@
               severity: 'warning',
             },
             annotations: {
-              message: 'Ceph cluster is in warning state',
+              message: 'Storage cluster is in warning state',
+              detailed_message: 'Storage cluster is in warning state',
```

Comment:
- Shouldn't it be `description` instead of `detailed_message`, like above?

- Can we make the description more detailed than keeping it same as message?

Reply:
Ack